### ISSUE TITLE
Improve UI

### ISSUE THREAD

**Issue by pnijhara** [AMU-Code-Squad/food-up#29]
Currently, the project uses bootstrap3 as the CSS framework!
- Need custom CSS for further UI improvement
- Need proper fonts
- Need custom js

**Comment by 4rtmelly** [comment]
Hi ! I don't know if this project is still active but I would like to work on this issue, is it possible ?

**Comment by pnijhara** [comment]
Hi, @4rtmelly this project is not actively maintained by anyone. But, yes if you want to work on this issue, you are free to do so. I'll try to review the PR.

**Comment by 4rtmelly** [comment]
Alright, however when I'm running the app with nodemon app.js on the localhost 3000 I get the website but I can't see articles and other stuff, is it normal ?

**Comment by pnijhara** [comment]
Yes! it is normal not to see any articles on the webpage because you need to set up a database locally on your own. After that app fetches the data from `seed.js` and show you the articles.

**Comment by 4rtmelly** [comment]
Okay ! Thanks a lot :) I'm gonna try to work on this

**Comment by Rohitchaudhary13** [comment]
Hey!, i would like to work on this project

**Comment by pnijhara** [comment]
@Rohitchaudhary13 If this is not related to HacktoberFest, then sure you can take over this issue.

**Comment by Rohitchaudhary13** [comment]
@pnijhara when i'm running the app with nodemon app.js the server is working but I'm not seeing any webpage on localhost 3000, is it normal?

**Comment by prithvi009** [comment]
hi, can I work on this issue?

**Comment by trakshan-mishra** [comment]
Hello, can I work on this issue?

**Comment by Ravi-Rathod143** [comment]
Hey, I would like work on this issue?

**Comment by ItsDee19** [comment]
hi @pnijhara , I have better UI designs. Can you assign me this issue?

**Comment by prakharsatyam** [comment]
Hey i'd love to work on this issue is it still required right now?
